apps/system/utils/README.md

```diff
@@ -103,15 +113,262 @@ This command supports sleeping of shell. This has an argument which shows sleepi
 The *CONFIG_DISABLE_SIGNALS* should not be enabled to use this.
 
 ### date
+This command prints the date. This has an argument which sets system time and date information.
+```
+TASH>>date --help
+
+Usage: date
+   or: date [-s FORMAT]
+Display, or Set system time and date information
+
+Options:
+ -s FORMAT     Set system time in the given FORMAT
+               FORMAT: MMM DD HH:MM:SS YYYY
+                'month', 'day', 'hour':'minute':'second' 'year'
+                Example: Apr 21 10:35:22 1991
+
+TASH>>date
+Jan 01 00:00:09 2010
+
+TASH>>date -s Feb 27 14:41:05 2018
+TASH>>date
+Feb 27 14:41:06 2018
+```
+Enable *CONFIG_ENABLE_DATE* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> date
+```
 
 ### dmesg
-### getenv / setenv / unsetenv
+This command shows all debug messages.
+```
+TASH>>dmesg
+s5j_sflash_init: FLASH Quad Enabled
+uart_register: Registering /dev/console
+uart_register: Registering /dev/ttyS0
+uart_register: Registering /dev/ttyS1
+uart_register: Registering /dev/ttyS2
+uart_register: Registering /dev/ttyS3
+uart_register: Registering /dev/ttyS4
+i2c_uioregister: Registering /dev/i2c-0
+i2c_uioregister: Registering /dev/i2c-1
+```
+Enable *CONFIG_ENABLE_DMESG* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> dmesg
+```
+The *CONFIG_SYSLOG* and *CONFIG_RAMLOG_SYSLOG* should be enabled to use this.
+
+### getenv/setenv/unsetenv
+These commands are related with setting or getting the environment variables.  
+getenv prints the environment variable. If there is no environment name, all environments are printed.   
```

Comment:
The ```**getenv**``` prints all of environment variables. If specific environment name is given, this prints the value of given envionment.